### PR TITLE
olya_heads_legs

### PR DESCRIPTION
**Pull request by olyapetryshyn**
…main.py.

### CODE REVIEW

**Review by LSikyr — DISMISSED** [review]

**Review by iPavliv — DISMISSED** [review]

File: main.py [hunk 1]
```diff
@@ -1,5 +1,19 @@
 """ToDo."""
 from tasks.kyu8 import example
+from tasks.kyu8 import heads_legs
+
+
+def run_heads_legs():
+    """
+    Runs heads_legs function.
+    """
+    heads = int(input("Enter the amount of heads: "))
```
Comment (ement06) [inline main.py]: Add check (try...except)

Comment (olyapetryshyn) [reply main.py]: fixed.

**Review by ement06 — DISMISSED** [review]

**Review by khadai — DISMISSED** [review]

File: main.py [hunk 2]
```diff
@@ -1,12 +1,22 @@
...
+    try:
+        heads = int(input("Enter the amount of heads: "))
+        legs = int(input("Enter the amount of legs: "))
+        while heads not in range(0, 1001) or legs not in range(0, 1001):
+            print("Bad input! Values must be positive and not greater than 1000.")
+            heads = int(input("Enter the amount of heads: "))
+            legs = int(input("Enter the amount of legs: "))
+    except ValueError:
+        print("Wrong input!")
+    print(heads_legs(heads, legs))
```
Comment (OlyaKhomyn) [inline main.py]: Consider calling the function inside a try-block because there is a chance you will call it with incorrect parameters

Comment (olyapetryshyn) [reply main.py]: fixed.

**Review by ement06 — APPROVED** [review]

**Review by ement06 — APPROVED** [review]

**Review by OlyaKhomyn — APPROVED** [review]

**Review by khadai — APPROVED** [review]

**Pull request merged**